data_validation/combiner.py

```diff
@@ -89,7 +90,11 @@ def generate_report(
         print(documented.compile())
 
     result_df = client.execute(documented)
-    result_df.validation_status.fillna(consts.VALIDATION_STATUS_FAIL, inplace=True)
+    result_df.status.fillna(consts.VALIDATION_STATUS_FAIL, inplace=True)
+
+    if filter_status:
```

Comment:
I'm not entirely sure this logic belongs in the combiner.  The status' are really more related to what is being served not what is being combined, and in that sense would make more sense in the result handler.

This would potentially also allow a result handler to make more tactical decisions, such as logging all results but only displaying/distributing the filtered results.

let me know what you think

Reply:
okay, sounds reasonable. then should we filter the result in `TextResultHandler#print_formatted_()` or `TextResultHandler#execute()`?

Reply:
okay, sounds reasonable. then should we filter the result in `TextResultHandler#print_formatted_()` or `TextResultHandler#execute()`?